code/modules/food_and_drink/snacks.dm

```diff
@@ -249,6 +250,60 @@
 		..()
 		src.add_topping(0)
 
+/obj/item/reagent_containers/food/snacks/pizza/fresh
+    name = "fresh pizza"
+    desc = "A cheesy pizza pie with thick tomato sauce."
+    icon_state = "cheesy"
+    sliced_icon = "cheesy-slice"
+	
```

Comment:
i thought this was a pizza ball and i got excited... sad

Reply:
no pizzaball was the working name for the meat ball pizza,  perhaps one day pizza will join the third dimension and become a sphere. To be honest there seems to be quite the demand for it.